oxd-server/src/main/java/org/xdi/oxd/server/op/GetAccessTokenByRefreshTokenOperation.java

```diff
@@ -43,6 +49,9 @@ public IOpResponse execute(GetAccessTokenByRefreshTokenParams params) throws Exc
             tokenClient.setExecutor(getHttpService().getClientExecutor());
             final TokenResponse tokenResponse = tokenClient.execRefreshToken(scopeAsString(params), params.getRefreshToken(), rp.getClientId(), rp.getClientSecret());
             if (tokenResponse != null) {
+                if (tokenResponse.getStatus() == 401) {
```

Comment:
401 means unauthorized. Are you sure refresh token is invalid and not client authentication? I guess we have to be careful here.

Reply:
Yes, it is 401.

Reply:
Please remove `if` statement. We don't need 401 check. Simply `handleInvalidRefreshToken(tokenResponse);`, if error type is there then we should process it. Also we don't want to depend on this 401 code because sometime oxauth returns 400 too.